deployment/cfn/worker.py

```diff
@@ -446,11 +457,34 @@ def get_cloud_config(self):
                 '    permissions: 0440\n',
                 '    owner: root:mmw\n',
                 '    content: |\n',
-                '      /dev/xvdf /opt/rwd-data\text4\tdefaults,nofail,discard\t0 2',  # NOQA
+                '      /dev/xvdf /opt/rwd-data\text4\tdefaults,nofail,discard\t0 2\n',  # NOQA
+                '\n',
+                'rsyslog:\n',
+                '  - $DefaultNetstreamDriverCAFile /etc/papertrail-bundle.pem # trust these CAs\n',
+                '  - $PreserveFQDN off\n',
+                '  - $ActionSendStreamDriver gtls # use gtls netstream driver\n',
+                '  - $ActionSendStreamDriverMode 1 # require TLS\n',
+                '  - $ActionSendStreamDriverAuthMode x509/name # authenticate by hostname\n',
+                '  - $ActionSendStreamDriverPermittedPeer *.papertrailapp.com\n',
+                '  - $ActionResumeInterval 10\n',
+                '  - $ActionQueueSize 100000\n',
+                '  - $ActionQueueDiscardMark 97500\n',
+                '  - $ActionQueueHighWaterMark 80000\n',
+                '  - $ActionQueueType LinkedList\n',
+                '  - $ActionQueueFileName papertrailqueue\n',
+                '  - $ActionQueueCheckpointInterval 100\n',
+                '  - $ActionQueueMaxDiskSpace 2g\n',
+                '  - $ActionResumeRetryCount -1\n',
+                '  - $ActionQueueSaveOnShutdown on\n',
+                '  - $ActionQueueTimeoutEnqueue 2\n',
+                '  - $ActionQueueDiscardSeverity 0\n',
+                '  - "*.*  @@', Ref(self.papertrail_host), ':', Ref(
+                    self.papertrail_port), '"\n',
+                'rsyslog_filename: 22-mmw-papertrail.conf\n',
                 '\n',
                 'runcmd:\n',
                 '  - cat /etc/fstab.rwd-data >> /etc/fstab\n',
-                '  - mount -t ext4 /dev/xvdf /opt/rwd-data && initctl emit rwd-ready\n',  # NOQA
+                '  - mount -t ext4 /dev/xvdf /opt/rwd-data && docker restart $(docker ps -q)\n',  # NOQA
```

Comment:
It looks like the RWD service has a name: `mmw_mmw-rwd_1`. I think that it would be more precise to restart just that vs. any container running on boot. We don't have any other running containers now, but if we ever add others, this approach could have unintended consequences.